### PR TITLE
Move config docs to guides

### PR DESCRIPTION
**Pull request by chizhg**
**What this PR does, why we need it**:
Move config docs to guides

### CODE REVIEW

File: guides/REAME.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Guides
+
+This folder contains documents for how to setup the CI/CD flow for a Knative
+project with the existing tools/scripts provided in this repository.
+
+- [How to setup Prow](./prow_setup.md)
```
Comment (chaodaiG) [inline guides/REAME.md]: I think this could be more verbose, like CI/CD flows for a new repo should be here

File: guides/REAME.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+- [How to setup CI/CD flows for a new repo with helper scripts](../scripts)
```
Comment (chaodaiG) [inline guides/REAME.md]: Isn't this for how to write test scripts?

File: guides/REAME.md (hunk 1 above)
Comment (chizhg) [reply guides/REAME.md]: Changed to `How to setup Prow and CI/CD flows for a new repo`

File: guides/REAME.md (hunk 2 above)
Comment (chizhg) [reply guides/REAME.md]: Changed to `How to implement your own test/release scripts with the helper scripts`

**Comment by chaodaiG** [comment]
Thank you!

/lgtm

**Pull request merged**